### PR TITLE
[CoinEX] Adding coinex to tentacles

### PR DESCRIPTION
**Pull request by nitrocon**

### CODE REVIEW

**Comment by Herklos** [comment]
Hi @nitrocon! Nice work

**Comment by nitrocon** [comment]
how to fix that? @Herklos

**Comment by Herklos** [comment]
> how to fix that? @Herklos

I'll try to relaunch the CI

**Comment by Herklos** [comment]
Let me just test it locally then we can merge it 😉

**Comment by Herklos** [comment]
@nitrocon it would be awesome if you create a new `community_tested` page with CoinEX at https://exchanges.octobot.info/ using https://github.com/Drakkar-Software/OctoBot-Docs/tree/exchange/stable 🙏

File: Trading/Exchange/coinex/coinex_exchange.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        real_trades_history = []
+
+        for exchange_manager in interfaces.get_exchange_managers(bot_api=bot_api,
+                                                             independent_backtesting=independent_backtesting):
+           if trading_api.is_trader_existing_and_enabled(exchange_manager):
+               if trading_api.is_trader_simulated(exchange_manager):
+                simulated_trades_history += trading_api.get_trade_history(exchange_manager, symbol, since, as_dict)
+               else:
+                real_trades_history += trading_api.get_trade_history(exchange_manager, symbol, since, as_dict)
+        return real_trades_history, simulated_trades_history
```
Comment (Herklos) [inline Trading/Exchange/coinex/coinex_exchange.py:71]: This code is not coinEx related right? I think you should remove it from this exchange implementation.

File: Trading/Exchange/coinex_websocket_feed/coinex_websocket.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        Feeds.CANDLE: cryptofeed_constants.CANDLES,
+    }
+
+    @classmethod
+    def get_name(cls):
+        return 'coinex'
+
+    @classmethod
+    def get_feed_name(cls):
+        return cryptofeed_constants.COINEX
```
Comment (Herklos) [inline Trading/Exchange/coinex_websocket_feed/coinex_websocket.py:36]: It seems that [cryptofeed](https://github.com/bmoscon/cryptofeed) doesn't support COINEX for now.  OctoBot raises an `<class 'AttributeError'>: module 'cryptofeed.defines' has no attribute 'COINEX'` at websocket startup using this. You should remove this websocket implementation while it's not supported by cryptofeed or implement your own websocket.

Comment (nitrocon) [reply Trading/Exchange/coinex_websocket_feed/coinex_websocket.py:36]: allright, PR to community_tested: https://github.com/Drakkar-Software/OctoBot-Docs/pull/55
websocket and code (did not work anyway) will be removed, thanks :)
But anyhow, I dont get the trade_history

Comment (Herklos) [reply Trading/Exchange/coinex_websocket_feed/coinex_websocket.py]: Thanks!

**Review by Herklos — APPROVED** [review]
Thanks for the changes

**Pull request merged**